grails-app/conf/ImagesClientResources.groovy

```diff
@@ -21,8 +21,8 @@ modules = {
         resource url: [plugin: "images-client-plugin", dir: 'js/audiojs', file: 'audio.min.js']
     }
 
-    viewer {
-        dependsOn 'jquery', 'leaflet', 'leaflet_draw', 'font-awesome'
+    image_viewer {
+        dependsOn 'jquery', 'bootstrap', 'leaflet', 'leaflet_draw', 'font-awesome'
```

Comment:
OK, I think the aim for the future should be to have a javascript component that can be used standalone in any web basedclient. It does not make sense that is only available as a grails plugin. 
Once you see the image gallery integrated with leaflet you'll see that there is no reason why this component could not be use by someone else besides the ALA.